### ISSUE TITLE
After deleting a default flow the model cannot be saved

### ISSUE THREAD

**Issue by falko** [camunda/camunda-modeler#368]
Tested in today's nightly

**Comment by falko** [comment]
I had to first make it an normal flow and then I was able to delete it

**Comment by falko** [comment]
Here is an example to reproduce. Just delete the Default Sequence Flow and try saving the model. Switching to the XML tab doesn't work either. I assume that the attribute on the Gateway is not deleted, which makes the model invalid.

```xml
<?xml version="1.0" encoding="UTF-8"?>
<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Definitions_1" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="1.3.0-nightly">
  <bpmn:process id="Process_1" isExecutable="false">
    <bpmn:startEvent id="StartEvent_1">
      <bpmn:outgoing>SequenceFlow_0ilci7k</bpmn:outgoing>
    </bpmn:startEvent>
    <bpmn:exclusiveGateway id="ExclusiveGateway_16b71dj" default="SequenceFlow_0d45nw9">
      <bpmn:incoming>SequenceFlow_0ilci7k</bpmn:incoming>
      <bpmn:outgoing>SequenceFlow_0d45nw9</bpmn:outgoing>
    </bpmn:exclusiveGateway>
    <bpmn:sequenceFlow id="SequenceFlow_0ilci7k" sourceRef="StartEvent_1" targetRef="ExclusiveGateway_16b71dj" />
    <bpmn:endEvent id="EndEvent_1cc26bx">
      <bpmn:incoming>SequenceFlow_0d45nw9</bpmn:incoming>
    </bpmn:endEvent>
    <bpmn:sequenceFlow id="SequenceFlow_0d45nw9" sourceRef="ExclusiveGateway_16b71dj" targetRef="EndEvent_1cc26bx" />
  </bpmn:process>
  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_1">
      <bpmndi:BPMNShape id="_BPMNShape_StartEvent_2" bpmnElement="StartEvent_1">
        <dc:Bounds x="173" y="102" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="ExclusiveGateway_16b71dj_di" bpmnElement="ExclusiveGateway_16b71dj" isMarkerVisible="true">
        <dc:Bounds x="245" y="95" width="50" height="50" />
        <bpmndi:BPMNLabel>
          <dc:Bounds x="225" y="145" width="90" height="20" />
        </bpmndi:BPMNLabel>
      </bpmndi:BPMNShape>
      <bpmndi:BPMNEdge id="SequenceFlow_0ilci7k_di" bpmnElement="SequenceFlow_0ilci7k">
        <di:waypoint xsi:type="dc:Point" x="209" y="120" />
        <di:waypoint xsi:type="dc:Point" x="245" y="120" />
        <bpmndi:BPMNLabel>
          <dc:Bounds x="182" y="95" width="90" height="20" />
        </bpmndi:BPMNLabel>
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNShape id="EndEvent_1cc26bx_di" bpmnElement="EndEvent_1cc26bx">
        <dc:Bounds x="338" y="102" width="36" height="36" />
        <bpmndi:BPMNLabel>
          <dc:Bounds x="311" y="138" width="90" height="20" />
        </bpmndi:BPMNLabel>
      </bpmndi:BPMNShape>
      <bpmndi:BPMNEdge id="SequenceFlow_0d45nw9_di" bpmnElement="SequenceFlow_0d45nw9">
        <di:waypoint xsi:type="dc:Point" x="295" y="120" />
        <di:waypoint xsi:type="dc:Point" x="338" y="120" />
        <bpmndi:BPMNLabel>
          <dc:Bounds x="271.5" y="95" width="90" height="20" />
        </bpmndi:BPMNLabel>
      </bpmndi:BPMNEdge>
    </bpmndi:BPMNPlane>
  </bpmndi:BPMNDiagram>
</bpmn:definitions>
```

**Comment by nikku** [comment]
Fixed with next patch release `v1.2.2`.
